assets/product-details.css

```
@@ -0,0 +1,115 @@
+}
+.buybox__product-options {
+  margin-bottom: 48px;
+}
+.buybox__product-price {
+  margin-bottom: 12px;
+}
+.buybox__product-cart {
+  margin-bottom: 24px;
+}
+.buybox__product-transport {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)